### PR TITLE
Selectores css

### PR DESCRIPTION
**Pull request by JenniferCastiblanco**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
- @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
\ No newline at end of file
+* @david31982 @holgiosalos @JuanDelahozMiranda @manuelq12 @valeryibarra
```
Comment (manuelq12) [inline .github/CODEOWNERS]: EOF

File: cypress.json [hunk 2]
```diff
@@ -1 +1,6 @@
-{}
+{
+    "video": false,
+    "screenshotOnRunFailure": false,
+    "defaultCommandTimeout": 20000,
+    "responseTimeout": 20000
+}
```
Comment (manuelq12) [inline cypress.json]: EOF

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            "node"
+        ]
+    },
+    "include": [
+        "cypress/**/*.ts"
+    ],
+    "exclude": [
+        "node_modules/"
+    ]
+}
```
Comment (manuelq12) [inline tsconfig.json]: EOF

**Review by manuelq12 — COMMENTED** [review]
Can you please add a screenshot with the green build of buy-shirt.spec.ts

File: cypress/integration/buy-shirt.spec.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+    cy.get("#SubmitLogin > span").click();
+    cy.get(".cart_navigation > .button > span").click();
+    cy.get("#cgv").click();
+    cy.get(".cart_navigation > .button > span").click();
+    cy.get(".bankwire").click();
+    cy.get("#cart_navigation > .button > span").click();
+    cy.get(".cheque-indent > .dark").should("have.text", "Your order on My Store is complete.");
+    cy.screenshot("Buy a t-shirt");
```
Comment (manuelq12) [inline cypress/integration/buy-shirt.spec.ts]: @JenniferCastiblanco That line is not required, what i meant is to add the screenshot of the test passed as a comment, in that way we can validate the execution of the test

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/integration/buy-shirt.spec.ts (hunk 4 above)
Comment (manuelq12) [reply cypress/integration/buy-shirt.spec.ts]: Something like this
Screen Shot 2022-05-09 at 4 31 13 PM

Comment (JenniferCastiblanco) [reply cypress/integration/buy-shirt.spec.ts]: Screenshot_1

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**